packages/zent/src/form/shared.tsx

```diff
@@ -154,7 +154,7 @@ export type IFormFieldProps<Value> = IFormFieldPropsBase<Value> &
 export type IFormComponentProps<
   Value,
   Props,
-  OmitKeys extends keyof IFormFieldPropsBase<Value> = never
+  OmitKeys extends string = ''
```

Comment:
why not 'never' ?

Reply:
it provides intellisense for available props